package.json

```diff
@@ -22,19 +22,23 @@
     "raw-loader": "^0.5.1",
     "to-string-loader": "^1.1.4",
     "typescript": "~1.8.9",
-    "typings": "~1.0.3",
+    "typings": "~1.3.2",
     "webpack": "^1.12.9",
     "webpack-dev-server": "^1.14.0",
-    "webpack-merge": "^0.8.4"
+    "webpack-merge": "^0.14.1"
   },
   "dependencies": {
-    "@angular/common": "2.0.0-rc.4",
-    "@angular/compiler": "2.0.0-rc.4",
-    "@angular/core": "2.0.0-rc.4",
-    "@angular/http": "2.0.0-rc.4",
-    "@angular/platform-browser": "2.0.0-rc.4",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.4",
-    "@angular/router": "3.0.0-beta.2",
+    "@angular/common": "2.0.0-rc.5",
+    "@angular/compiler": "2.0.0-rc.5",
+    "@angular/core": "2.0.0-rc.5",
+    "@angular/http": "2.0.0-rc.5",
+    "@angular/compiler-cli": "0.5.0",
+    "@angular/platform-server": "2.0.0-rc.5",
+    "@angular/forms": "0.3.0",
+    "@angular/platform-browser": "2.0.0-rc.5",
+    "typescript": "^1.9.0-dev",
```

Comment:
here I need your guidance, the things that are new here apart from the rc5 update are : 
-   the inclusion of the forms module ( consider that when this seed was initially created the same functionality was included in common... I'd argue it needs to be in any seed
-   the inclusion of the compiler-cli module - to allow AoT compilation and it's dependent on typscript which is also included
 - the inclusion of platform-server for use of AoT compiler via node

so please guide me whether to omit some or all of these, the rc5 ver change obviously needs to to be kept..,